spec/nest_thermostat_spec.rb

```diff
@@ -40,10 +49,13 @@ module NestThermostat
     end
 
     it "sets away status" do
+      # Freeze time so we always use the same time (makes VCR happy)
+      Timecop.freeze(Time.local(2015, 10, 27, 10, 5, 0))
```

Comment:
Do not use `Time.local` without zone. Use one of `Time.zone.local`, `Time.local.current`, `Time.local.in_time_zone`, `Time.local.utc`, `Time.local.getlocal`, `Time.local.iso8601`, `Time.local.jisx0301`, `Time.local.rfc3339`, `Time.local.to_i`, `Time.local.to_f` instead.